src/Interface/Modules/Render/ViewScene.cc

```diff
@@ -299,6 +302,15 @@ void ViewSceneDialog::addScreenshotButton()
   addToolbarButton(screenshotButton);
 }
 
+//--------------------------------------------------------------------------------------------------
+void ViewSceneDialog::addBugReportButton() {
```

Comment:
SCIRun standard is all open braces on own newline

Reply:
Sorry, my ide auto formats sometimes